README.md

```diff
@@ -165,13 +166,13 @@ Datahike's query engine and most of its codebase come from
 DataScript, Datahike would not have been possible. Differences to Datomic with
 respect to the query engine are documented there.
 
-## When should I pick what?
+## When to Choose Datahike vs. Datomic vs. DataScript
```

Comment:
Why is "Choose" upper case?

Reply:
Because that's how I got taught to capitalize English titles... I guess we have to decide for a style, at least per document. In the README it looks like they were mixed even before I changes this title

Reply:
Alright, then let's go with your uppercase way and adjust all the relevant titles.

Reply:
In all of the files or just the README?

Reply:
Just the README for now.

Reply:
ok

Reply:
Done